doc/getting_started/index.rst

```diff
@@ -116,8 +95,7 @@ and more on how ``west`` helps manage multiple repositories.
 Install Python Dependencies
 ***************************
 
-Next, install additional Python packages required by Zephyr in a shell or
-``cmd.exe`` prompt:
+Install Python packages required by Zephyr:
```

Comment:
"Less is more" yet cleaning and deleting stuff is a difficult job that rarely gets the credit it deserves. Impressive dedication @mbolivar.

This being said, pip deserves a bit more introduction. So here's a suggested addition:

> [`pip`](https://pypi.org/project/pip) is Python's package installer. It first tries to re-use packages and package dependencies already installed by your operating system. If the requested versions are not found then it downloads them from the Internet and installs them in the .local/ subdirectory of your HOME. Note the package versions requested by Zephyr may conflict with other versions requested by other software, in that case you may want to look at [virtualenv](https://virtualenv.pypa.io/en/latest/)

pip is extremely important for Zephyr. It manages a large and crucial part of Zephyr dependencies and can puzzle users when it fails. This seems fortunately rare, except and as previous observed by @mbolivar,  Zephyr's `requirements.txt` files (there's another one in ci-tools, to be merged [one way](https://github.com/zephyrproject-rtos/zephyr/pull/15992) or the other) is very picky which could clash with other software. For security reasons this guide should not hide that pip downloads a ton of code from random places on the Internet. Right now there's not even a link to pip's documentation!
See some previous discussion at: https://lists.zephyrproject.org/g/users/message/1308 and elsewhere.

Reply:
I think you're right that this is important information that should be present in our documentation. I want to balance this with brevity in a 1,2,3 style getting started guide, however, where we are intentionally glossing over many details that are important in favor of getting people going.

How would you feel about a short user guide on pip, linked to from the getting started guide?

Reply:
I agree with @marc-hb, pip deserves at least a paragraph in our documentation. I think @mbolivar 's suggestion of a linked to short guide hits the sweet spot between brevity and thoroughness.

Reply:
While we seem to agree zero information and zero link about pip is not enough, I don't have enough pip information for a standalone page either. If my 4 sentences above are too much then take only the first half or maybe even less (an introduction and pointer to another, standalone page wouldn't be much shorter anyway)

Reply:
Given this feedback, I'm going to put Marc's content into a footnote, and a link to https://packaging.python.org/tutorials/installing-packages/ inline in the text.